rules/parser.y

```diff
@@ -235,6 +237,28 @@ extra_labels_opts  :
                      { $$ = true }
                    ;
 
+vector_matching    :
+                     { $$ = nil }
+                   | GROUP_OP '(' label_list ')'
+                     { 
+                       var err error
+                       $$, err = newVectorMatching("", $3, nil)
+                       if err != nil { yylex.Error(err.Error()); return 1 }
+                     }
+                   | GROUP_MOD GROUP_OP '(' label_list ')'
```

Comment:
That doesn't make sense if GROUP_MOD is supplied, as it can only do a one-to-one matching then.